libs/ballot-interpreter-vx/test/helpers/replace_snapshot_values.ts

```diff
@@ -0,0 +1,59 @@
+// TODO: Import this helper from `@votingworks/test-utils`. There is some
+// dependency issue currently preventing this.
```

Comment:
See #2945. I wanted to unblock myself. Will add a reference to the issue into the code.